tiny_gltf.h

```diff
@@ -183,6 +183,8 @@ namespace tinygltf {
 #define TINYGLTF_DOUBLE_EPS (1.e-12)
 #define TINYGLTF_DOUBLE_EQUAL(a, b) (std::fabs((b) - (a)) < TINYGLTF_DOUBLE_EPS)
 
+#define TINYGLTF_USE_HUNTER
```

Comment:
Define this in the CMakeLists? You should also export all relevant defines using `target_compile_definitions(tinygltf PUBLIC <DEFINE_NAME>)` so that downstream projects get it.